clients/linux/src/app.rs

```diff
@@ -603,7 +603,17 @@ impl LbApp {
                     d.close();
                     let acctscr = &lb.gui.account;
                     acctscr.sidebar.tree.set_name(&id, &name);
-                    lb.messenger.send(Msg::RefreshSyncStatus);
+
+                    match lb.core.file_by_id(meta.id) {
```

Comment:
I made a variable for the id, so we can just use it here:
```suggestion
                    match lb.core.file_by_id(id) {
```